parser/ParseBasicExpressions.hpp

```diff
@@ -392,6 +400,13 @@ class ParseFunctionCall : public ParseExpression {
   }
 
   /**
+   * @return Whether this function call contains the DISTINCT operator.
```

Comment:
So, how about we use `property` instead of `operator` when referring `distinct`?